### DISCUSSION TITLE
Welcome to the posit::conf(2023) workshop on MLOps with vetiver 🏺

### DISCUSSION THREAD

**Discussion by juliasilge** [posit-conf-2023/vetiver#4] [Announcements]
## 👋 Welcome!
  We’re using GitHub Discussions as a place to connect during [our posit::conf(2023) workshop](https://posit-conf-2023.github.io/vetiver). We hope that you:
  * ask questions you’re wondering about, especially if you don't want to ask aloud
  * share ideas
  * engage with your fellow workshop participants

Check out the [code of conduct](https://posit.co/code-of-conduct/) ❤️  for this conference and our workshop.

To get started, comment below with an introduction of yourself and tell us about what kind of role you have, where you've come from, and what you are looking forward to this week!

**Comment by rkb965** [comment]
So happy to be here! I'm Rebecca Butler from SoCal, and I am a biostatistician who sometimes works with predictive models. I have been looking forward to learning about {vetiver} since I first heard whispers about this package. I work in an academic-adjacent environment where funding mechanisms are such that long-term follow up of model performance can be hard to do, so learning how to monitor with as little active human time as possible is really exciting to me. Happy meet you all!

**Comment by isabelizimm** [comment]
Hi all 👋 I'm Isabel, one of your TAs for the day. I'm so excited to be here and meet everyone!

**Reply by vasantm** [reply]
quick question - what's the posit cloud link for this workshop?

**Reply by isabelizimm** [reply]
We will be using Workbench + Connect. The link is [pos.it/class](http://pos.it/class)!

**Comment by blairj09** [comment]
👋 I'm James Blair and I work at Posit as a product manager for cloud integrations. I'm a long-time R user and advocate and have been with Posit for 5 years. I live in Utah (near-ish to Julia) with my wife, 5 kids, and golden-doodle puppy. I'm excited to be here as one of the TAs for this course!

**Comment by vasantm** [comment]
👋 Hi, I am Vasant Marur - currently a Senior Data Scientist at Merck ([www.merck.com](https://www.merck.com/)). I build pipelines to analyze image data - and use deep learning to classify images. I use R for data processing, statistical analysis and visualizations. I'm based in Boston! so if you want to know more about that or the kind of work can be done in Big Pharma, chat me up! Excited to be here and learn more about vetiver with you all

**Comment by aaronball85** [comment]
I'm Aaron Ball and am excited to learn with you all today! I do a broad range of data analytics work at BYU-Idaho and BYU-Pathway Worldwide. I'm hopeful to learn more about deploying and maintaining R-built models with {vetiver}. Making the models available as REST APIs on the Azure cloud, or published to Azure Databricks would be of particular interest. If others have this interest or experience in these areas, I'd love to visit with and learn from you!

**Reply by blairj09** [reply]
Aaron! It would be great to chat - we're doing a lot of work with Databricks right now and it would be helpful to understand your existing workflows and desired future state.
